velox/dwio/common/SelectiveByteRleColumnReader.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/common/SelectiveColumnReaderInternal.h"
+
+namespace facebook::velox::dwio::common {
+
+class SelectiveByteRleColumnReader : public SelectiveColumnReader {
+ public:
+  SelectiveByteRleColumnReader(
+      std::shared_ptr<const dwio::common::TypeWithId> requestedType,
+      dwio::common::FormatParams& params,
+      velox::common::ScanSpec& scanSpec,
+      const TypePtr& type)
+      : SelectiveColumnReader(
+            std::move(requestedType),
+            params,
+            scanSpec,
+            type) {}
+
+  bool hasBulkPath() const override {
+    return false;
+  }
+
+  void getValues(RowSet rows, VectorPtr* result) override {
```

Comment:
Probably should keep non-trivial methods definition in cpp file to reduce build time